### PR TITLE
https://issues.redhat.com/browse/ACM-8481

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

File: observability/use_observability.adoc [hunk 1]
```diff
@@ -208,6 +208,13 @@ The total number of clusters that are exceeding or meeting the targeted _service
 +
 The error budget for the past seven or 30-day period, the remaining downtime, and trend are displayed.
 
+[#viewing-ocpvirt-dashboards]
+=== Viewing the {ocp-virt-short} dashboard
+//will you remind me what HCO is?
```
Comment (dockerymick) [inline observability/use_observability.adoc]: @moadz will you remind me what HCO is?

Comment (sradco) [reply observability/use_observability.adoc]: HCO is the operator that deploys and monitors the rest of the OpenShift Virtualization operators. It is a short for Hyperconverged Cluster Operator, https://github.com/kubevirt/hyperconverged-cluster-operator

Comment (moadz) [reply observability/use_observability.adoc]: I suspect it's this, but not sure https://github.com/kubevirt/hyperconverged-cluster-operator

Comment (moadz) [reply observability/use_observability.adoc]: Thanks @sradco :)

**Review by moadz — APPROVED** [review]
LGTM

File: observability/use_observability.adoc [hunk 2]
```diff
@@ -208,6 +208,13 @@ The total number of clusters that are exceeding or meeting the targeted _service
...
+You can view the {ocp-virt} dashboard to see comprehensive insights for each cluster with the {ocp-virt-short} operator installed. The state of the operator is displayed, which is determined by active {ocp-virt-short} alerts and the HCO conditions of the operator. Additionally, you view the number of running virtual machines and the operator version for each cluster.
```
Comment (dockerymick) [inline observability/use_observability.adoc]:
```suggestion
You can view the {ocp-virt} dashboard to see comprehensive insights for each cluster with the {ocp-virt-short} operator installed. The state of the operator is displayed, which is determined by active {ocp-virt-short} alerts and the conditions of the Hyperconverged Cluster Operator. Additionally, you view the number of running virtual machines and the operator version for each cluster. 
```

File: observability/use_observability.adoc (hunk 1 above)
Comment (dockerymick) [inline observability/use_observability.adoc]:
```suggestion

```

**Comment by dockerymick** [comment]
@moadz @sradco thanks to you both for the feedback! Reaching out to my team for a peer review

File: release_notes/whats_new.adoc [hunk 3]
```diff
@@ -69,6 +69,8 @@ For other Application topics, see link:../applications/app_management_overview.a
 
 * You can enable the _Fleet_ switch to view the potential risks for updating your managed clusters. See, link:../observability/insights_intro.adoc#update-risks[Viewing update risk predictions]. 
 
+* You can now view the {ocp-virt} dashboard to see comprehensive insights for each cluster with the {ocp-virt-short} operator installed. See, link:../observability/use_observability.adocviewing-ocpvirt-dashboards[Viewing the {ocp-virt-short} dashboard].
```
Comment (dockerymick) [inline release_notes/whats_new.adoc]:
```suggestion
* You can now view the {ocp-virt} dashboard to see comprehensive insights for each cluster with the {ocp-virt-short} operator installed. See, link:../observability/use_observability.adoc#viewing-ocpvirt-dashboards[Viewing the {ocp-virt-short} dashboard].
```

File: observability/use_observability.adoc [hunk 4]
```diff
@@ -208,6 +208,12 @@ The total number of clusters that are exceeding or meeting the targeted _service
 +
 The error budget for the past seven or 30-day period, the remaining downtime, and trend are displayed.
 
+[#viewing-ocpvirt-dashboards]
+=== Viewing the {ocp-virt-short} dashboard
+You can view the {ocp-virt} dashboard to see comprehensive insights for each cluster with the {ocp-virt-short} operator installed. The state of the operator is displayed, which is determined by active {ocp-virt-short} alerts and the conditions of the Hyperconverged Cluster Operator. Additionally, you view the number of running virtual machines and the operator version for each cluster.
```
Comment (dockerymick) [inline observability/use_observability.adoc:214]:
```suggestion

You can view the {ocp-virt} dashboard to see comprehensive insights for each cluster with the {ocp-virt-short} operator installed. The state of the operator is displayed, which is determined by active {ocp-virt-short} alerts and the conditions of the Hyperconverged Cluster Operator. Additionally, you view the number of running virtual machines and the operator version for each cluster. 
```

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**